docs/user/alerting/action-types/webhook.asciidoc

```diff
@@ -73,3 +73,16 @@ Password::  An optional password. If set, HTTP basic authentication is used. Cur
 Webhook actions have the following properties:
 
 Body::      A json payload sent to the request URL.
+
+[float]
+[[Example of a JSON payload]]
+--
+
+{
+  "short_description": "{{context.rule.name}}",
+  "description": "{{context.rule.description}}",
+  ...
+}
+--
+For the variables that can be defined in an alert, refer to {ref}/painless-watcher-condition-context.html[Watcher condition text].
```

Comment:
As I know the alerting action types aren't used in the Watcher plugin. So this doc improvement could be useful for alerting, but we should remove any Watcher mentioning here like `[Watcher condition text].` 
I think that having the JSON example body is great, but in addition to this we should mention that it could be an other Content-Types as well.

Reply:
removed